src/App.css

```
@@ -60,75 +61,99 @@
 
 
 /* Remove Body Scroll */
-
 ::-webkit-scrollbar {
     width: 0px;
     /* remove scrollbar space */
     background: transparent;
     /* optional: just make scrollbar invisible */
 }
-
-@-moz-document url-prefix() {
-    html,
-    body {
-        overflow: hidden !important;
-    }
-}
-
-
 /* Remove Body Text Highlight */
 
 ::selection {
     color: none;
     background: none;
 }
-
-
 /* For Mozilla Firefox */
 
 ::-moz-selection {
     color: none;
     background: none;
 }
 
+
+@-moz-document url-prefix() {
+    html,
+    body {
+        overflow: hidden !important;
+    }
+}
+html,
 body {
     background-color: #282c34;
-    min-height: 100vh;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
-    font-size: calc(10px + 2vmin);
-    color: white;
+    color: #fff;
+    font-family: var(--main-font);
 }
-
-
-/* Check over after merge */
-
-.App {
-    text-align: center;
+h1, h2, h3, p, a, div, input{
+    color: #fff!important; 
```

Nit: !important property spacing
```suggestion
    color: #fff !important; 
```

---

Remove at later date.